src/pages/guides/eqp/v1/help.md

```diff
@@ -9,7 +9,7 @@ Use these resources if you need additional help.
 
 **Slack:**
 
--  workspace: [Magento Community Engineering](https://opensource.magento.com/slack)
+-  workspace: [Magento Community Engineering](https://developer.adobe.com/open/magento/slack)
```

Comment:
The name of the workspace is **Magento Open Source**  magentocommeng.slack.com 

![image](https://github.com/AdobeDocs/commerce-marketplace/assets/44397551/d384423d-0614-4008-8fa0-3c4fd917b320)